streetlevel/streetview/panorama.py

```diff
@@ -339,3 +339,11 @@ class ArtworkLink:
     """The pano ID which the annotation links to."""
     link_text: LocalizedString
     """The link text."""
+
+@dataclass
+class StreetLabel:
+    """A label that floats in the air showing you the names of streets in official road coverage."""
+    name: LocalizedString
+    """The street's name."""
+    angles: List[float]
+    """Angles that this street name appears at, in radians."""
```

Comment:
I think you should clarify that this is the yaw specifically.

Reply:
Oops, hope I've used the resolve conversation button there properly. Maybe that's clarified now.